Framework/jmaqs-selenium/src/test/java/com/magenic/jmaqs/selenium/SeleniumConfigTest.java

```diff
@@ -49,6 +49,28 @@ public void getWebsiteBase() {
     Assert.assertTrue(website.equalsIgnoreCase("http://magenicautomation.azurewebsites.net/"));
   }
 
+  /**
+   * Hub Url.
+   */
+  @Test
```

Comment:
We have test categories now that you could add:
`@Test(groups = TestCategories.Selenium)`